src/modules/catalog/catalog-service.js

```diff
@@ -82,7 +83,8 @@ angular.module('anol.catalog')
                         self.firstLetters.push(firstLetter);
                         sortedLayers[firstLetter] = {
                             'layers':  [_layer],
-                            'title': firstLetter
+                            'title': firstLetter,
+                            'localeTitle': voca.latinise(firstLetter)
```

Comment:
I think `localeTitle` suggest some kind of translated value here. Maybe it would be better to call this `latinizedTitle`?

Reply:
I looked into voca and does not convert `Ä` to `Ae` but only `A`. I think this is no major issue. But it would be nicer if all `Ä`s would be grouped together.

Reply:
I just noticed that this is just the first letter. So this is a way to do this.

Reply:
`Ä`s, 'Ö's and ' Ü's  are all still grouped together so it actually does not matter for the filter because the groups with accents always come after the group without umlaut.

Reply:
I will change the name